sensors/test_deps.go

```diff
@@ -65,7 +63,7 @@ func getWarmingUpLidar() *inject.Camera {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
 	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-		return camera.Properties{}, nil
+		return camera.Properties{SupportsPCD: true}, nil
```

Comment:
We shouldn't need to change this now right? We should only change this once we do the work referred to here: https://github.com/viamrobotics/viam-cartographer/pull/232/files#diff-255612532e71d064ed5e0725da530352d9e537d7410e071ace9aa15b3f7f8c36R51 right?

Same goes to all the other PropertiesFunc changes in this file.

Reply:
yes and no - I think it will be good to have it, since they should have had it in the first place in order to be a valid lidar configuration. But you are correct in that we are not currently checking their properties

Reply:
please remove / comment out all the `SupportsPCD: true` if they are not needed in this PR